exoctk/exoctk_app/app_exoctk.py

```diff
@@ -437,7 +437,7 @@ def contam_visibility():
 
             # Make plot
             title = form.targname.data or ', '.join([form.ra.data, form.dec.data])
-            pG, pB, dates, vis_plot, table = vpa.using_gtvt(str(form.ra.data), str(form.dec.data), form.inst.data)
+            pG, pB, dates, vis_plot, table = vpa.using_gtvt(str(form.ra.data), str(form.dec.data), form.inst.data.split(' ')[0])
```

Comment:
@jaymedina why do we need the split here?

Reply:
`form.inst.data` actually returns `NIRCam [Filter]` for the NIRCam options, but my function `using_gtvt` only wants the instrument name, so that was my quick way to fix the issue. This only happens for NIRCam, because users have the option to select `NIRCam F322W2` or `NIRCam F444W` for the Contam Visibility tool.

Reply:
Awesome -- Thanks for the explanation!